tutorials/preprocessing/plot_60_maxwell_filtering_sss.py

```diff
@@ -1,45 +1,197 @@
+#      :func:`~mne.preprocessing.maxwell_filter` in order to prevent bad
+#      channel noise from spreading.
+#
+# To see the effect, we can plot the data before and after SSS / Maxwell
+# filtering.
+
+raw.pick(['meg']).plot(duration=2, butterfly=True)
+raw_sss.pick(['meg']).plot(duration=2, butterfly=True)
+
+###############################################################################
+# Notice that channels marked as "bad" have been effectively repaired by SSS,
+# eliminating the need to perform :ref:`interpolation <tut-bad-channels>`.
+# The heartbeat artifact has also been substantially reduced.
+#
+# The :func:`~mne.preprocessing.maxwell_filter` function has parameters
+# ``int_order`` and ``ext_order`` for setting the order of the spherical
+# harmonic expansion of the interior and exterior components; the default
+# values are appropriate for most use cases. Additional parameters include
+# ``coord_frame`` and ``origin`` for controlling the coordinate frame ("head"
+# or "meg") and the origin of the sphere; the defaults are appropriate for most
+# studies that include digitization of the scalp surface / electrodes. See the
+# documentation of :func:`~mne.preprocessing.maxwell_filter` for details.
+#
+#
+# Spatiotemporal SSS (tSSS)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# An assumption of SSS is that the measurement volume (the spherical shell
+# where the sensors are physically located) is free of electromagnetic sources.
+# The thickness of this source-free measurement shell should be 4-8 cm for SSS
+# to perform optimally. In practice, there may be sources falling within that
+# measurement volume; these can often be mitigated by using Spatiotemporal
+# Signal Space Separation (tSSS) [2]_. tSSS works by looking for temporal
+# correlation between components of the internal and external subspaces, and
+# projecting out any components that are common to the internal and external
+# subspaces. The projection is done in an analogous way to
+# :ref:`SSP <tut-artifact-ssp>`, except that the noise vector is computed
+# across time points instead of across sensors.
+#
+# To use tSSS in MNE-Python, pass a time (in seconds) to the parameter
+# ``st_duration`` of :func:`~mne.preprocessing.maxwell_filter`. This will
+# determine the "chunk duration" over which to compute the temporal projection.
+# The chunk duration effectively acts as a high-pass filter with a cutoff
+# frequency of :math:`\frac{1}{\mathtt{st\_duration}}~\mathrm{Hz}`; this
+# effective high-pass has some important consequences:
+#
+# - In general, larger values of ``st_duration`` are better (provided that your
+#   computer has sufficient memory) because larger values of ``st_duration``
+#   will have a smaller effect on the signal.
+#
+# - It is best to choose a chunk duration that evenly divides your data length,
+#   so that the effective high-pass filtering is the same for all spans of your
+#   data.
```

Comment:
I'm not sure we really need to recommend this. If you have multiple subjects and/or runs, the only way to really satisfy this is to have the duration vary across subjects and runs, which is probably a worse tradeoff than having just the last chunk for each subject/run possibly be different (at least for typical duration recordings and `st_duration` choices).

Perhaps instead we should just make a note that the last window could use a longer duration (at worse just shy of `2 * st_duration`)?

Reply:
LMK if you're happy with the revised wording.